docs/samples/browser-plugin-meetings/app.js

```diff
@@ -427,19 +427,27 @@ function refreshCaptcha() {
 meetingsListElm.onclick = (e) => {
   selectedMeetingId = e.target.value;
   const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+  const selectedMeetingType = createMeetingSelectElm.options[createMeetingSelectElm.selectedIndex].innerText;
+  console.log("MEETING TYPE AAAA: ",selectedMeetingType);
 
   if (meeting && meeting.passwordStatus === 'REQUIRED') {
     meetingsJoinPinElm.disabled = false;
     verifyPasswordElm.disabled = false;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
-  else if (meeting && meeting.passwordStatus === 'UNKNOWN') {
+  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && (selectedMeetingType !== "One to One Meeting" && selectedMeetingType!=="Email" && selectedMeetingType!=="PERSON ID" ))) {
     meetingsJoinPinElm.disabled = true;
     verifyPasswordElm.disabled = true;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
+  else if(meeting && (meeting.passwordStatus === 'UNKNOWN' && (selectedMeetingType === "One to One Meeting" && selectedMeetingType==="Email" && selectedMeetingType==="PERSON ID" ))) {
```

Comment:
And here we can do the opposite. So when `selectedMeetingType !== "SIP"`

Reply:
Same as above.

Reply:
@Shreyas281299 Updated code